### PR TITLE
Build and publish with Python 3.10 in GitHub actions.

### LINKED ISSUE #185: The wheel for Python 3.10 is not available
Hello! Could you please build a wheel of pywinpty 1.1.4 for Python 3.10 and upload it to PyPI? Thank you!

### PR DESCRIPTION
**Pull request by joseph-roitman**
Fixes #185

I test ran the GitHub actions on my [fork](https://github.com/joseph-roitman/pywinpty/actions) and everything seems to work fine.
I also built pywinpty in a local Miniconda environment and successfully installed the built wheel in a clean Python 3.10 installation.

### CODE REVIEW

File: .github/workflows/linux_sdist.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                PYTHON_VERSION: ["3.9"]
+                PYTHON_VERSION: ["3.10"]
```
Comment (ccordoba12) [inline .github/workflows/linux_sdist.yml]: Why this change? This is not related to building wheels.

**Review by ccordoba12 — COMMENTED** [review]
@joseph-roitman, thanks a lot for your help with this! I left a small comment for you, otherwise looks good to me.

File: .github/workflows/linux_sdist.yml (hunk 1 above)
Comment (joseph-roitman) [reply .github/workflows/linux_sdist.yml]: I thought it would be ideal to keep things up to date, but I can revert this change if you prefer.

**Comment by andfoy** [comment]
@joseph-roitman, thanks for the help with this one!
